README.md

```diff
@@ -126,18 +126,20 @@ tx.add_transfer(
     recipient="cosmos103l758ps7403sd9c0y8j6hrfw4xyl70j4mmwkf", amount=387000
 )
 tx.add_transfer(recipient="cosmos1lzumfk6xvwf9k9rk72mqtztv867xyem393um48", amount=123)
-pushable_tx = tx.get_pushable()
 
 
-# Optionally submit the transaction using your preferred method.
-# This example uses the httpx library.
-import httpx
+import requests
```

Comment:
I prefer to keep endorsing the `httpx` library rather than `requests`.

Reply:
What is the advantage to use httpx? I would say requests is by far the most common library for http requests

Reply:
`httpx` does async (as well as sync) calls.

And yeah I'm free to endorse whichever library I want, am I not? :wink:

Reply:
I never said something else, lol. That makes sense. I just noticed that while 
```python
rpc_url = "https://rpc.cosmos.network/"
pushable_tx = json.dumps({
                "jsonrpc": "2.0",
                "id": 1,
                "method": "broadcast_tx_sync", # Available methods: broadcast_tx_sync, broadcast_tx_async, broadcast_tx_commit
                "params": {
                    "tx": tx_bytes
                }
              })

r = requests.post(rpc_url, data=pushable_tx)
``` works httpx wasn't:
```python
rpc_url = "https://rpc.cosmos.network/"
pushable_tx = {
                "jsonrpc": "2.0",
                "id": 1,
                "method": "broadcast_tx_sync", # Available methods: broadcast_tx_sync, broadcast_tx_async, broadcast_tx_commit
                "params": {
                    "tx": tx_bytes
                }
              }

r = httpx.post(rpc_url, data=pushable_tx)
```

Which would give me:
```
{
  "jsonrpc": "2.0",
  "error": {
    "code": -32700,
    "message": "Parse error. Invalid JSON",
    "data": "error unmarshaling request: invalid character 'j' looking for beginning of value"
  }
}
```

Well, seems like I have to see what's wrong

Reply:
The httpx example is missing a `json.dumps` isn't it?

Reply:
You're absolutely right.. .And that's how I initially made it until Pycharm complained that it got a str instead of a dict. Well, it works now